S4TF_Tutorial_1.ipynb

```diff
@@ -95,7 +95,7 @@
         "\n",
```

Comment:
In the introduction above, maybe changing "Welcome to this Colab" to "Welcome to this notebook" would read more cleanly, given that Colab is mentioned later in the sentence.